rewrite-java/src/main/java/org/openrewrite/java/tree/JavaType.java

```diff
@@ -302,6 +302,23 @@ public static Class build(int flagsBitMap,
             }
         }
 
+        /**
+         * Can be used to find a class from the flyweights by fully-qualified name. If more than one class exists for
+         * the name, the first one is returned. This method will not return matches for fully-qualified classes that
```

Comment:
So this method can never find a class that accepts a type parameter? So it couldn't find `java.util.ArrayList`?
That sounds like it could be a source of bugs

Reply:
yeah, I am rethinking if this is a good idea at all. The original intent was to resolve a JavaType.Class from a JavaType.FullyQualified via its fully qualified name.